packages/framework/ini-files/config-specs.ini

```diff
@@ -2935,6 +2935,11 @@ use PACKAGE-ENABLES|NO-PACKAGE-ENABLES
 
 use COMMON_SPACK_TPLS
 
+opt-set-cmake-var TPL_BLAS_LIBRARY_DIRS     STRING FORCE : ${OPENBLAS_ROOT|ENV}/lib
+opt-set-cmake-var TPL_BLAS_LIBRARIES        STRING FORCE : ${OPENBLAS_ROOT|ENV}/lib/libopenblas.a;-L${OPENBLAS_ROOT|ENV}/lib;-lgfortran;-lgomp;-lm
+opt-set-cmake-var TPL_LAPACK_LIBRARY_DIRS   STRING FORCE : ${OPENBLAS_ROOT|ENV}/lib
+opt-set-cmake-var TPL_LAPACK_LIBRARIES      STRING FORCE : ${OPENBLAS_ROOT|ENV}/lib/libopenblas.a;-L${OPENBLAS_ROOT|ENV}/lib;-lgfortran;-lgomp;-lm
```

Comment:
We could get around this if we set a more-general variable in the container, correct?  Meaning, if openblas' module set `BLAS_ROOT` instead, we could probably not need these?

Reply:
yes we could do something like that. The other thing we could do is add links in the bin directory of openblas install so you get libblas and liblapack as links pointing to libopenblas

Reply:
Spoke with Joe at standup today and decided to leave it as is for now.